colossalai/inference/config.py

```diff
@@ -393,3 +393,64 @@ class ModelShardInferenceConfig:
     use_cuda_kernel: bool = False
     use_spec_dec: bool = False
     use_flash_attn: bool = False
+
+
+@dataclass
+class GenerationParams:
+    """
+    Param for diffusion
+    """
+
+    # prompt: Union[str, List[str]] = None # NOTE no sure if we should add this param
+    prompt_2: Optional[Union[str, List[str]]] = None
+    prompt_3: Optional[Union[str, List[str]]] = None
+    height: Optional[int] = None
+    width: Optional[int] = None
+    num_inference_steps: int = 28
+    timesteps: List[int] = None
+    guidance_scale: float = 7.0
+    negative_prompt: Optional[
+        Union[str, List[str]]
+    ] = ""  # NOTE(@lry89757) in pixart default to "", in sd3 default to None, we set "" here, but the best solution is change the check_inputs of each forward fuction
```

Comment:
For type hint defined as `Optional[...]` in python, we usually use `None` as its default value.

Reply:
trivial: And you might want to place relatively long comment to a different line (not inline) so that the formatter won't reformat and place it to a longer line with statements

Reply:
Thx!